aws_lambda_powertools/event_handler/util.py

```diff
@@ -71,12 +71,14 @@ def extract_origin_header(resolver_headers: Dict[str, Any]):
     return resolved_header
 
 
-def validate_openapi_security_parameters(
+def _validate_openapi_security_parameters(
     security: List[Dict[str, List[str]]],
     security_schemes: Optional[Dict[str, "SecurityScheme"]],
```

Comment:
```suggestion
    security_schemes: Optional[Dict[str, "SecurityScheme"]] = None,
```

If the type annotation is correct, you're missing a `None`